doc/ld-mapping.md

```diff
@@ -94,15 +94,15 @@ Example:
 
 ### Data preparation
 
-The datasets were very well prepared by FOEN. All structural data is available as its own CSV file which makes it extremely simple to expose everything as RDF. After first tests it was decided to introduce one additional column to the structural data: A field name which still looks well after it is [URL encoded](https://en.wikipedia.org/wiki/Percent-encoding). This is optional but it makes generated HTTP URIs in the RDF data much more readable. This work was done by the data owner for each row in the CSV.
+The datasets were very well prepared by FOEN. All structural data was available as its own CSV file which made it extremely simple to expose everything as RDF. After first tests it was decided to introduce one additional column to the structural data: A field name which still looks well after it is [URL encoded](https://en.wikipedia.org/wiki/Percent-encoding). This is optional but it makes generated HTTP URIs in the RDF data much more readable. This work was done by the data owner for each row in the CSV.
 
-Next to the structural data there is one base table. This table contains the "normalized" data according to the specification of the dataset. Each reference to another table is done using the field name mentioned above as key. This makes it very easy to reference the structural data and the generated URI for each measure looks human readable as well.
+Next to the structural data there is one base table. This table contains the "normalized" data according to the specification of the dataset. Each reference to another table is done using the field name mentioned above as key. This makes it very easy to reference the structural data, and the generated URI for each measure is human readable as well.
```

Comment:
"This makes it very easy to create RDF out of it which can then be referenced from the base table shown before." - The sentence seems very vague to me. Can we be more precise?